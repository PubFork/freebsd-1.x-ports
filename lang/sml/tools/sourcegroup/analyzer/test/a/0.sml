structure B = struct val x=4 end
structure C = struct val z=5 end
structure D = struct val y=5 end
