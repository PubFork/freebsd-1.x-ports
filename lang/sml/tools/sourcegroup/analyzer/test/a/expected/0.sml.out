source sml "a/0.sml"
  export structure B C D ;

